lib/driver.js

```diff
@@ -497,6 +508,10 @@ class IosDriver extends BaseDriver {
       }
       logger.debug('Waiting for initial webview');
       await this.navToInitialWebview();
+      // we don't need UIAutoClient and Instruments any more,
+      // should stop them to avoid 'Abnormal Instruments error' due to the command timeout.
+      await this.stopUIAutoClient();
```

Comment:
i don't think we want to do this except in the case of a real device. With a sim, we still have a good instruments session!